src/test/java/pages/HomePage.java

```diff
@@ -3,17 +3,36 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
 
-public class HomePage extends BasePage {
+public class HomePage extends pages.BasePage {
 
-    By userAvatarIcon = By.cssSelector("img.avatar");
+    //Locators
+    @FindBy(css = "[class='home active']")
+    WebElement homeButton;
+    @FindBy(css = "[class='queue']")
+    WebElement currentQue;
+    @FindBy(css = "[class='songs']")
+    WebElement allSongsButton;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 


```
    @FindBy(css = "[class='home active']")
    private WebElement homeButton;
    @FindBy(css = "[class='queue']")
    private WebElement currentQue;
    @FindBy(css = "[class='songs']")
    private WebElement allSongsButton;
    private By userAvatarIcon = By.cssSelector("img.avatar");
```